### ISSUE TITLE
Design improvement: icons and text visibility

### ISSUE THREAD

**Issue by panyann** [Gabboxl/FluentWeather#82]
Hi,
First of all - thank you for the app - it's great.

However in my opinion, there is a room for the design improvements:
![obraz](https://github.com/user-attachments/assets/3ec7bfc7-e58e-46c5-b5b2-6fa8bef841da)

If you could look into it, would be great 👍

**Comment by Gabboxl** [comment]
Hi, you're right. I always hated this too actually, but I never found time to explore more design options to avoid this... Will look into this this week.

**Comment by Steperman5692** [comment]
what about adding shadows on the text?

**Comment by panyann** [comment]
I am a web developer, so for me the simplest solution would be to make the semi transparent elements dark instead of light.
Second way would be to make a dark semi transparent layer between the elements and the wallpaper, to make all wallpapers darker.
Or maybe divide wallpapers to light and dark and display UI differently for both cases.

But as always it's the author who knows what's the best for their app 🙂

**Comment by Gabboxl** [comment]
> Or maybe divide wallpapers to light and dark and display UI differently for both cases.

yeah, I was going for something like that... i would like to try doing a sort of "dynamic" style for elements that tries to adapt to the wallpaper's luminosity
